packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+@mixin bone-width($width) {
+  &.pt-skeleton-bone-#{$width} {
+    width: $width * 1%;
+  }
+}
+
+.pt-skeleton-bone {
```

Comment:
`.pt-skeleton` will be the parent thing later, so I'd go for `.pt-skeleton-line` or `.pt-skeleton-bone`... Not sure if it's nice to keep going with the metaphor